src/Providers/NStack.php

```diff
@@ -299,22 +345,21 @@ protected function getCredentials()
 
         // Missing credentials. Abort.
         if (!empty($missingCredentials)) {
-            throw new MissingCredentialsException(sprintf('Missing credentials: %s', implode($missingCredentials, ', ')));
+            throw new MissingCredentialsException(sprintf('Missing credentials: %s', implode($missingCredentials, ', ')), 500);
         }
 
-        return $this->credentials;
+        return $this->appCredentials;
     }
 
     /**
-     * Request translations from NStack
+     * request
      *
-     * @author Morten Rugaard <moru@nodes.dk>
-     * @date   13-10-2015
-     * @access public
-     * @param  string $locale
-     * @param  string $platform
-     * @return mixed
-     * @throws \Nodes\Translate\Exception\TranslateRequestException
+     * @author Casper Rasmussen <cr@nodes.dk>
+     * @access protected
+     * @param $locale
+     * @param $platform
```

Comment:
Missing what type the variable is. Remember it if can be almost everything, like `string`, `array` etc. you should use the `mixed` type instead of going `string|array|integer`.